src/components/shared/SearchInput.css

```diff
@@ -65,6 +65,7 @@
 }
 
 .search-field input:focus {
+  background: var(--theme-toolbar-background-hover);
```

Comment:
Couple of suggestions:
The background is too flush with controls that are before and after the input field. Could you please add a padding-inline-start: 10px; for the input field so the cursor is not at the very edge of the input field. Also could you add 10px padding-left to the '.summary' element so there is a little bit of space between the summary and the input field (but only when summary has any results).

Reply:
Yeah this does not seem to work. I think it's fine to add it to not just :focus styling but to the overall .search-field input styling. I would add:
```
padding-inline-start: 10px;
padding-inline-end: 10px;
margin-inline-end: 10px;
```
What it does is it makes sure that there is always space between text and background borders: text inside input field and text for the number of results.